src/cudamatrix/cu-matrix-lib.h

```diff
@@ -26,6 +26,7 @@
 #include "cudamatrix/cu-matrix.h"
 #include "cudamatrix/cu-sp-matrix.h"
 #include "cudamatrix/cu-tp-matrix.h"
+#include "cudamatrix/cu-sparse-matrix.h"
```

Comment:
Thanks.
Regarding the this->data_ -- at some point I got in the habit of doing it
because we had issues with some compiler version not resolving data members
of base classes without the this->.  I think the fact that it's a template
may also be relevant.    I don't remember which compiler and which version
though.
I did not realize that the includes had to be sorted alphabetically; I'll
try to make sure they are sorted in future.
Dan


On Wed, Aug 19, 2015 at 5:34 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/cudamatrix/cu-matrix-lib.h
> <https://github.com/kaldi-asr/kaldi/pull/83#discussion_r37485317>:
>
> > @@ -26,6 +26,7 @@
> >  #include "cudamatrix/cu-matrix.h"
> >  #include "cudamatrix/cu-sp-matrix.h"
> >  #include "cudamatrix/cu-tp-matrix.h"
> > +#include "cudamatrix/cu-sparse-matrix.h"
>
> Includes not sorted alphabetically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/83/files#r37485317>.
>